torchao/quantization/quant_api.py

```diff
@@ -466,3 +481,7 @@ def int8_dynamic_activation_int8_semi_sparse_weight():
     """
     from torchao.dtypes import SemiSparseLayoutType
     return int8_dynamic_activation_int8_weight(layout_type=SemiSparseLayoutType())
+
+
+if TORCH_VERSION_AFTER_2_5:
```

Comment:
A n00b qn: it wasn't super clear to me why these functions are used during unpickling

but I trust that you added them because they appeared in the error message :)

Reply:
yeah that's correct, these functions are serialized as well when we do dynamic quantization